cmd/runner/main.go

```diff
@@ -324,6 +338,23 @@ func (h *handler) parsePortfolio(ctx context.Context, taskID task.ID, req *task.
 	return nil
 }
 
+func countLines(path string) (int, error) {
```

Comment:
depending on what this is used for, we probably want to account for the CSV header row? Only really matters for small N

Reply:
good thought - I'll subtract 1. It's only used as a metric we can include in the metadata to help users quickly differentiate portfolios from one another using the most basic metrics.